copter/source/docs/zigzag-mode.rst

```diff
@@ -30,3 +30,21 @@ When flown manually ZigZag uses :ref:`Loiter mode's <loiter-mode>`  parameters.
 
 When autonomously flying from one side of the field to the other, the :ref:`Auto mode <auto-mode>` parameters are used.  For example the top speed can be configured with :ref:`WPNAV_SPEED <WPNAV_SPEED>`..
 
+Automatic Operation
+===================
+
+.. note:: this function must be enabled with :ref:`ZIGZ_AUTO_ENABLE<ZIGZ_AUTO_ENABLE>` = 1, and rebooted, before any parameters are visible and the feature enabled.
+
+After the field's side points are registered, if you have an RC switch configured with ``RCx_OPTION`` = 83 (ZigZag Auto Mode), you can automatically cover the field side to side at a programmable distance interval, also enabling a sprayer, if equipped.
+
+.. image:: ../../../images/zigzag-auto.png
+     :target: ../_images/zigzag-auto.png
+
+To use:
+
+- Fly to one side of the field and register the ZigZagWP A, as above, then fly to ZigZagWP B and set it.
+- Then switch the ZigZag Auto Mode switch you had setup. The vehicle will proceed as shown in the diagram, automatically enabling a sprayer, if setup using ``SERVOx_FUNCTION`` = 22 and :ref:`ZIGZ_SPRAYER<ZIGZ_SPRAYER>` = 1 (enable sprayer during ZigZag Auto).
```

Comment:
Instead of specifying SERVOx_FUNCTION = 22 (sprayer pump) we should just link to the crop sprayer setup wiki page.  https://ardupilot.org/copter/docs/sprayer.html

Reply:
done